index.html

```diff
@@ -7526,6 +7526,27 @@ <h5>Protocol and Payload Bindings</h5>
           </p>
         </section>
       </section>
+
+      <section id="profile-relationship">
+        <h4>Relationship to Profiles</h4>
+        <p>
+          The Web of Things also define a profiling mechanism in [[WOT-PROFILES]] that allows the reduction of
+          implementation effort. Profiles are built on top of existing Bindings by constraining the flexibility of TDs.
+          Such constraints can be bindings, semantic contexts, link relations, security schemes and discovery
+          mechanisms. As there is less flexibility, the implementation effort to achieve out-of-the-box interoperability
+          is reduced.
```

Comment:
"Less flexibility" is a bit of negative way of framing the constraints that Profiles impose.

This sentence implies that it is possible to achieve out-of-the-box interoperability between Consumers and Things without Profiles, if you just put enough effort into your implementation. I don't think this is actually true, because with an unbounded set of protocols and payload formats etc. it is _never_ possible to guarantee that a given Consumer can communicate with all possible Things.

I understand that you don't like to say that Profiles "guarantee interoperability" since there are arguably levels of interoperability (e.g. semantic interoperability) which are currently still left open, but they certainly provide _additional_ interoperability guarantees over using bindings alone.